django/core/checks/security/base.py

```diff
@@ -97,6 +108,19 @@
     id='security.W021',
 )
 
+W022 = Warning(
+    "You have not set the SECURE_REFERRER_POLICY setting. Without this, your "
+    "site will not send a Referrer-Policy header. You should consider enabling "
+    "this header to protect user privacy.",
+    id="security.W022"
+)
+
+E001 = Error(
+    "You have set the SECURE_REFERRER_POLICY setting to an invalid value. Valid "
+    "values are: {}".format(', '.join(REFERRER_POLICY_VALUES)),
+    id="security.E001"
+)
```

Comment:
Use the `hint` parameter for listing valid values, e.g.
```python
E023 = Error(
    'You have set the SECURE_REFERRER_POLICY setting to an invalid value.',
    hint='Valid values are: {}.'.format(', '.join(sorted(REFERRER_POLICY_VALUES))),
    id='security.E023',
)
```

Also, the current convention is to not have independent numbering for warnings and errors, so use `E023` instead of `E001`.

Reply:
These checks are not documented in `docs/refs/checks.txt` and `W001` also needs updating..